classes/classes/Creature.as

```diff
@@ -2203,7 +2203,7 @@ package classes
 		{
 			switch (gender) {
 				case GENDER_NONE:   return caps ? mf("Genderless", "Fem-genderless") : mf("genderless", "fem-genderless");
-				case GENDER_MALE:   return caps ? mf("Male", "Dickgirl")             : mf("male", "dickgirl");
+				case GENDER_MALE:   return caps ? mf("Male", biggestTitSize() > 1 ? "Shemale" : "Femboy")             : mf("male", biggestTitSize() > 1 ? "shemale" : "femboy");
```

Comment:
Please use `Appearance.BREAST_CUP_A` or `BREAST_CUP_A` instead of a magic number.